assets/diagram-js-minimap.css

```diff
@@ -31,14 +31,13 @@
 }
 
 .djs-minimap:not(.open) .toggle {
-  width: 46px;
-  height: 46px;
-  line-height: 46px;
+  padding: 10px;

```

Comment:
This is not related to the translation issue, right?

Reply:
Yes it is. Fixed width does not work, otherwise you end up with an overflowing box and/or weird spacing. I replaced the line-height/height with padding for simplicity. 
If you want to maintain the 46px height, just keep the line-height/height and only set padding to left/right.

Reply:
All right. So we should never have had fixed width here.